### PR TITLE
Denazification

### PR DESCRIPTION
**Pull request by sam0x2b**

### CODE REVIEW

File: src/identifiers.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Identifiers follow the specification in [Unicode Standard Annex #31][UAX31] for
 * `foo`
 * `_identifier`
 * `r#true`
-* `Москва`
+* `Київ`
```
Comment (bjorn3) [inline src/identifiers.md:21]: Rather than replacing Moscow with Kyiv, I think it should be replaced with an entirely neutral term.

Comment (sam0x2b) [reply src/identifiers.md]: ok, lets make it "Lemon"

**Review by bjorn3 — APPROVED** [review]
Lemon is a good neutral term. I don't believe I have write permission, so someone else will have to merge.

**Comment by ehuss** [comment]
I'm going to close this PR. I am very sympathetic for the events going on, and the Rust lang org has publicly stated its support for the people of Ukraine. However, this kind of PR and language isn't appropriate, and this isn't a forum for making a statement.

**Pull request closed without merging**